app/assets/javascripts/components/features/notifications/components/notification.jsx

```diff
@@ -51,7 +54,9 @@ class Notification extends React.PureComponent {
           <div className='notification__favourite-icon-wrapper'>
             <i className='fa fa-fw fa-retweet' />
           </div>
-
+          <div className='account__avatar-boost'>
+            <Avatar src={notification.getIn(['account', 'avatar'])} staticSrc={notification.getIn(['account', 'avatar_static'])} size={20} />
+          </div>
```

Comment:
Since this is the same code in three different places, it seems like a good candidate for making a generic component and being DRY.

Is it possible to set the size in CSS rather than as a prop?